Artesian/Artesian.SDK/Service/Query/IMasQuery.cs

```diff
@@ -1,10 +1,16 @@
 ﻿// Copyright (c) ARK LTD. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for
 // license information. 
+using NodaTime;
+
 namespace Artesian.SDK.Service
 {
     interface IMasQuery<T>: IQuery<T>
     {
         T ForProducts(params string[] products);
+        T WithFillNull();
```

Comment:
All these methods but WithFillCustomValue should be in the IQuery<T> interface.

Only WithFillCustomValue should be specialized in IActualQuery and IMasQuery